src/TextDateInput/storybook/TextDateInput.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC, useState } from 'react'
+import { noop } from '../../utils/noop'
+import {
+  FullDateFormat,
+  TextDateInput,
+  TextDateInputProps,
+} from '../TextDateInput'
+
+export default {
+  title: 'Text Date Input',
+  component: TextDateInput,
+  argTypes: { onChange: { action: 'changed' } },
+}
+
+const Template: FC = (props: TextDateInputProps) => {
+  const [_value, setValue] = useState<FullDateFormat>({
```

Comment:
Why is it `_value`?

Reply:
This was left over from a previous approach where there was already a `value` defined, we can use `value` now though so I'll update this